gusto/core/configuration.py

```diff
@@ -269,6 +269,20 @@ class BoundaryLayerParameters(Configuration):
     mu = 100.                   # Interior penalty coefficient for vertical diffusion
 
 
+class Held_Suarez_Parameters(Configuration):
```

Comment:
To be consistent with the other objects, can we avoid the underscores?
```suggestion
class HeldSuarezParameters(Configuration):
```